tests/tuner/test_lr_finder.py

```diff
@@ -184,7 +184,7 @@ def __init__(self):
 
     assert lrfinder.suggestion() != 1e-3
     assert len(lrfinder.results["lr"]) == 100
-    assert lrfinder._total_batch_idx == 200
+    assert lrfinder._total_batch_idx == 199
```

Comment:
Why do we get a diff there ? Is the test deterministic ?

Reply:
> Is the test deterministic?

It is.

Seems to me like the previous `total_batch_idx` was wrong, this is because the name indicates an index but the previous value here (`200`) indicates a count.

So we could keep the old count behaviour (`200`) with a misleading variable name (`*_idx`)
or the old variable name (`*_idx`) with the fixed index behaviour (`199`).

For the LR finder in particular, seems like this attribute is just used for debugging, so should be fine to change it.